client/app/vps/dashboard/vps-dashboard.controller.js

```diff
@@ -25,21 +25,35 @@ class VpsDashboardCtrl {
     }
 
     initLoaders () {
+        const zonesToDisplayUpgradeMessageTo = ["os-sbg1-006", "os-sbg1-010", "os-sbg1-011", "os-sbg1-012", "os-sbg1-013"];
+
         this.vps = this.ControllerHelper.request.getHashLoader({
             loaderFunction: () => this.VpsService.getSelectedVps(this.serviceName),
             successHandler: () => {
                 if (!this.vps.data.isExpired) {
                     this.loadIps();
                     this.hasAdditionalDiskOption();
                 }
+
+                const shouldDisplayUpgradeMessage = _(zonesToDisplayUpgradeMessageTo).includes(this.vps.data.zone);
+
+                if (shouldDisplayUpgradeMessage) {
+                    const url = "http://travaux.ovh.net/?do=details&id=32631";
+                    const textHtml = `<p>${this.$translate.instant("vps_dashboard_upgrade_1")}</p>
```

Comment:
Why not using a simply using a [`<oui-message>`](https://master.ui-kit.ovh/#!/oui-angular/message)? Instead of creating DOM Nodes right from the controller.

Reply:
Because @Netnix asked me to use `CloudMessage` last time 😅

Reply:
@antleblanc @FredericEspiau @Netnix 

You should use `<oui-message>` instead to homogenize with others managers :)

When CloudMessage will be in all manager, we will consider it :) #uniformisation

One more thing, personally, I don't like defining the HTML template in a controller, I prefer to keep the separation HTML / JS and avoid mixing everything.

Here, by defining all in the Javascript file, you don't take the benefits of angular (binding etc).

In the future, if we want to grep a specific tag in .html files, we should grep js file too ...  If this specific case, I think that we all understand, but if all of us do that, the code will quickly become a happy mess 🎉 

I don't like this method, for a lot a reasons, if you want, we can discuss about that :)

PS: You should add a title on the link to specify that it will be opened in a new window :)

See u!

Reply:
PS: This comment is of course contextualized.

If one day we use JSX, we will have made the choice to put the HTML templates in JS, my comment will be different. But in our case, we are not there, so let's keep a suitable structure :)

Reply:
@Netnix explained last time that `CloudMessage` actually uses a `oui-message` underneath. I'll use `oui-message` as I also prefer this solution but I don't want to hear complains about using `oui-message` this time.

Also, once again, I don't like the `title` attribute as W3C specifies that it's not recommended for accessibility purposes: https://www.w3.org/TR/html/dom.html#the-title-attribute